src/main/scala/org/opennetworkinsight/LDA.scala

```diff
@@ -23,21 +22,19 @@ object LDA {
       case Some(config) => {
 
         val logger = LoggerFactory.getLogger(this.getClass)
-        apacheLogger.getLogger("org").setLevel(Level.OFF)
-        apacheLogger.getLogger("akka").setLevel(Level.OFF)
+        Logger.getLogger("org").setLevel(Level.OFF)
+        Logger.getLogger("akka").setLevel(Level.OFF)
 
```

Comment:
The idea was to differentiate between the logger variable and Logger class. I guess it looks good anyways.